app/src/main/java/com/lorenzovainigli/foodexpirationdates/di/AppModule.kt

```diff
@@ -23,7 +24,8 @@ class AppModule {
             application,
             AppDatabase::class.java,
             "database.db"
-        ).fallbackToDestructiveMigration()
+        ).addMigrations(MIGRATION_3_4)
```

Comment:
Could you please add some unit tests for this migration [here](https://github.com/lorenzovngl/FoodExpirationDates/blob/main/app/src/androidTest/java/com/lorenzovainigli/foodexpirationdates/model/MigrationTest.kt)?